gundi_client/client.py

```diff
@@ -48,13 +48,13 @@ def __init__(self, **kwargs):
         self.cached_token_expires_at = datetime.min.replace(tzinfo=timezone.utc)
 
         self.max_retries = kwargs.get('max_http_retries', self.DEFAULT_CONNECTION_RETRIES)
-        transport = AsyncHTTPTransport(retries=self.max_retries)
+        transport = AsyncHTTPTransport(retries=self.max_retries, verify=settings.CDIP_ADMIN_SSL_VERIFY)
```

Comment:
Good catch. Looks like some settings set in the transport take precedence over settings set at the client level (but not all). I'll apply the same fix in the v2 client.
https://github.com/encode/httpx/discussions/1867

Reply:
PR with same fix in v2 client
https://github.com/PADAS/gundi-client/pull/22